dindctl

```diff
@@ -46,13 +46,14 @@ SUPERVISOR_EXTRA_MOUNTS=""
 DIST_MOUNTED="false"
 DIND_IMAGE="resin-supervisor-dind"
 PRELOADED_IMAGE=""
+OPTIMIZE="true"
```

Comment:
That's a nice idea, but I'd leave it for later as it involves a few more changes